assets/javascripts/discourse/templates/list/topic-list-item.raw.hbs

```diff
@@ -46,13 +55,20 @@
   </div>
```

Comment:
@merefield What does this closing div match? ``topic-details``? If so, what does the closing div on line 47 match? There are circumstances in which both will be present.

Reply:
I've reverted this file to the current HEAD